spacecmd/src/spacecmd/utils.py

```diff
@@ -370,11 +376,21 @@ def parse_time_input(userinput=''):
     logging.error(_N('Invalid time provided'))
     return None
 
-
-# Compares 2 package objects (dicts) and returns the newest one.
-# If the objects are the same, we return None
 def latest_pkg(pkg1, pkg2, version_key='version',
                release_key='release', epoch_key='epoch'):
+    """
+    Compares 2 package objects (dicts) and returns the newest one.
+
+        Parameters:
+            pkg1 (dict): A dictionary
+            pkg2 (dict): A dictionary
+            version_key (str): A string
+            release_key (str): A string
+            epoch_key (str): A string
+
```

Comment:
Here maybe we could add the semantics of the parameters version, release and epoch  (instead of just writing the type).